sdk/synapse/Azure.Analytics.Synapse.Artifacts/tests/LinkedServiceClientLiveTests.cs

```diff
@@ -39,21 +50,25 @@ await foreach (var expectedLinkedService in LinkedServiceClient.GetLinkedService
         [Test]
         public async Task TestCreateLinkedService()
         {
+            LinkedServiceClient client = CreateClient ();
+
             string linkedServiceName = Recording.GenerateName("LinkedSercive");
-            LinkedServiceCreateOrUpdateLinkedServiceOperation operation = await LinkedServiceClient.StartCreateOrUpdateLinkedServiceAsync(linkedServiceName, new LinkedServiceResource(new AzureDataLakeStoreLinkedService("adl://test.azuredatalakestore.net/")));
+            LinkedServiceCreateOrUpdateLinkedServiceOperation operation = await client.StartCreateOrUpdateLinkedServiceAsync(linkedServiceName, new LinkedServiceResource(new AzureDataLakeStoreLinkedService("adl://test.azuredatalakestore.net/")));
             LinkedServiceResource linkedService = await operation.WaitForCompletionAsync();
             Assert.AreEqual(linkedServiceName, linkedService.Name);
         }
 
         [Test]
         public async Task TestDeleteLinkedService()
         {
+            LinkedServiceClient client = CreateClient ();
+
             string linkedServiceName = Recording.GenerateName("LinkedSercive");
 
-            LinkedServiceCreateOrUpdateLinkedServiceOperation createOperation = await LinkedServiceClient.StartCreateOrUpdateLinkedServiceAsync(linkedServiceName, new LinkedServiceResource(new AzureDataLakeStoreLinkedService("adl://test.azuredatalakestore.net/")));
+            LinkedServiceCreateOrUpdateLinkedServiceOperation createOperation = await client.StartCreateOrUpdateLinkedServiceAsync(linkedServiceName, new LinkedServiceResource(new AzureDataLakeStoreLinkedService("adl://test.azuredatalakestore.net/")));
```

Comment:
Should this be an embedded URL?   Is it stable across platforms/environments?

Reply:
I don't have a `test.azuredatalakestore.net` setup, so I believe it's just a fake test URL. I believe it should be stable in other test env.